cmd/config_dirigent_dandelion_trace.json

```diff
@@ -2,6 +2,7 @@
   "Seed": 42,
 
   "Platform": "Dirigent-Dandelion",
+  "InvokeProtocol" : "http1",
```

Comment:
Maybe write httpVersion and choose between 1 and 2 as integer

Reply:
We support GRPC also, so we cannot name the parameter `httpVersion`.

Reply:
Ok this makes sense. Then you can discard this comment